### ISSUE TITLE
problems with trajectory_ makes the pkg not build-able by colcon in ROS2 humble (humble-devel)

### ISSUE THREAD

**Issue by javichisco** [pal-robotics/play_motion2#1]
ThirdParty/play_motion2/play_motion2/src/play_motion2/motion_planner.cpp:286:24: error: ‘struct moveit::planning_interface::MoveGroupInterface::Plan’ has no member named ‘trajectory_’; did you mean ‘trajectory’?
  286 |     if (!approach_plan.trajectory_.joint_trajectory.points.empty()) {
      |                        ^~~~~~~~~~~
      |                        trajectory

ThirdParty/play_motion2/play_motion2/src/play_motion2/motion_planner.cpp:291:21: error: ‘struct moveit::planning_interface::MoveGroupInterface::Plan’ has no member named ‘trajectory_’; did you mean ‘trajectory’?
  291 |   if (approach_plan.trajectory_.joint_trajectory.points.empty()) {
      |                     ^~~~~~~~~~~
      |                     trajectory

ThirdParty/play_motion2/play_motion2/src/play_motion2/motion_planner.cpp:295:45: error: ‘struct moveit::planning_interface::MoveGroupInterface::Plan’ has no member named ‘trajectory_’; did you mean ‘trajectory’?
  295 |   return perform_motion(info, approach_plan.trajectory_.joint_trajectory);
      |                                             ^~~~~~~~~~~
      |                                             trajectory

gmake[2]: *** [CMakeFiles/play_motion2.dir/build.make:90: CMakeFiles/play_motion2.dir/src/play_motion2/motion_planner.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:165: CMakeFiles/play_motion2.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2

**Comment by saikishor** [comment]
@javichisco which MoveIt version are you using? Can you tell us which ROS Distro are you using?

**Comment by javichisco** [comment]
ROS2 Humble
Ubuntu 20.04

Using play_motion2 1.1.1
Updated the last week

I changed trajectory_ with trajectory in the mentioned lines... and I was able to build the pkg with colcon.. (286, 291, 295)... in the motion planner cpp file... but I am not sure about the results and the behavior will be the same.

**Comment by Noel215** [comment]
Hi @javichisco,

I've tested it in a fresh `humble` docker and a new workspace, but I cannot reproduce the issue.

Also, for further testing I've installed the `ros-humble-tiago-simulation` package and I've run the simulation with `ros2 launch tiago_gazebo tiago_gazebo.launch.py`, and everything is working for me.

The version of MoveIt 2 I'm using is the latest: `2.5.5`.

Which version of MoveIt 2 are you using?

Also, note that the `humble` docker is an Ubuntu 22.04, not Ubuntu 20.04.
